src/Frontend/Core/Engine/Form.php

```diff
@@ -65,7 +70,7 @@ public function __construct(string $name, string $action = null, $method = 'post
      *
      * @throws Exception
      */
-    public function addButton($name, string $value, string $type = 'submit', string $class = null): \SpoonFormButton
+    public function addButton($name, $value, $type = 'submit', $class = null)
```

Comment:
Cause we need to update spoon library for that, but we can do that after this MR